Makefile

```diff
@@ -139,15 +137,7 @@ konk-operator-${GIT_VERSION}.tgz:
 %-${GIT_VERSION}.tgz:
 	${HELM} package helm-charts/$* --version ${GIT_VERSION}
 
-konk-service-operator-${GIT_VERSION}.tgz:
-	mkdir -p helm-charts/konk-service-operator/crds
-	cp -vR config/crd/bases/* helm-charts/konk-service-operator/crds/
-	cp -vR config/rbac helm-charts/konk-service-operator/
-	rm helm-charts/konk-service-operator/rbac/konk_editor_role.yaml
-	rm helm-charts/konk-service-operator/rbac/konk_viewer_role.yaml
-	${HELM} package helm-charts/konk-service-operator --version ${GIT_VERSION} --app-version ${GIT_VERSION}
-
-package: konk-operator-${GIT_VERSION}.tgz konk-${GIT_VERSION}.tgz konk-service-operator-${GIT_VERSION}.tgz konk-service-${GIT_VERSION}.tgz
+package: konk-operator-${GIT_VERSION}.tgz konk-${GIT_VERSION}.tgz konk-service-${GIT_VERSION}.tgz
```

Comment:
Looks good, but the rm `konkservice_editor_role.yaml` and `rm konkservice_viewer_role.yaml` need to be removed from  the `konk-operator-${GIT_VERSION}.tgz` target